Newsletter2Go/Export/view/frontend/layout/default.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Magento
```

Comment:
Remove useless comment (especially since we probably have our own license). Also that `@`tags are probably useless.